<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSecondHandBooks</class>
 <widget class="QWidget" name="CSecondHandBooks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="Tab_displayBooks">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>481</width>
     <height>561</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="schoolBooks">
    <attribute name="title">
     <string>教学用书类</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>6</y>
       <width>471</width>
       <height>521</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollArea_School">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>469</width>
        <height>519</height>
       </rect>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>-1</y>
         <width>381</width>
         <height>511</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="vl_School">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item>
         <widget class="QWidget" name="wdg_list_school" native="true">
          <property name="minimumSize">
           <size>
            <width>379</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="wdg_menu_school" native="true">
          <property name="minimumSize">
           <size>
            <width>379</width>
            <height>50</height>
           </size>
          </property>
          <widget class="QPushButton" name="pushButton_3">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>10</y>
             <width>341</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string>加载更多数据</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="noSchoolBooks">
    <attribute name="title">
     <string>非教学用书类</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>6</y>
       <width>471</width>
       <height>511</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollArea_NoSchool">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>469</width>
        <height>509</height>
       </rect>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>-1</y>
         <width>381</width>
         <height>511</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="vl_NoSchool">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item>
         <widget class="QWidget" name="wdg_list_noschool" native="true">
          <property name="minimumSize">
           <size>
            <width>379</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="wdg_menu_noschool" native="true">
          <property name="minimumSize">
           <size>
            <width>379</width>
            <height>50</height>
           </size>
          </property>
          <widget class="QPushButton" name="pushButton_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>341</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string>加载更多数据</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>其他非书籍类物品</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>6</y>
       <width>471</width>
       <height>511</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scorllArea_other">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>469</width>
        <height>509</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>509</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>-1</y>
         <width>381</width>
         <height>511</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="vl_Other">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item>
         <widget class="QWidget" name="wdg_list_other" native="true">
          <property name="minimumSize">
           <size>
            <width>379</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="wdg_menu_other" native="true">
          <property name="minimumSize">
           <size>
            <width>379</width>
            <height>50</height>
           </size>
          </property>
          <widget class="QPushButton" name="pushButton">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>341</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string>加载更多数据</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>搜索结果</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea_select">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>6</y>
       <width>471</width>
       <height>521</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrolArea_Select">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>469</width>
        <height>519</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>-1</y>
         <width>391</width>
         <height>521</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="vl_select">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item>
         <widget class="QWidget" name="wdg_list_select" native="true">
          <property name="minimumSize">
           <size>
            <width>379</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="wdg_menu_select" native="true">
          <property name="minimumSize">
           <size>
            <width>379</width>
            <height>50</height>
           </size>
          </property>
          <widget class="QPushButton" name="pushButton_4">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>10</y>
             <width>341</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string>加载更多数据</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QLineEdit" name="le_bookName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>您可以在这里输入书名来进行搜索</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea_4">
   <property name="geometry">
    <rect>
     <x>499</x>
     <y>146</y>
     <width>400</width>
     <height>521</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>0</height>
    </size>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollArea_Mybook">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>398</width>
      <height>519</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>0</y>
       <width>261</width>
       <height>521</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <widget class="QWidget" name="wdg_list_mybook" native="true"/>
      </item>
      <item>
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>20</y>
     <width>211</width>
     <height>100</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QPushButton" name="pb_sellBooks">
      <property name="text">
       <string>我要卖书</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_MyBooks">
      <property name="text">
       <string>加载我上架的所有物品</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_deleteAllMybooks">
      <property name="text">
       <string>一键下架所有物品</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>322</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pb_selectBooks">
      <property name="text">
       <string>搜索教学用书</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_select_otherbook">
      <property name="text">
       <string>搜索非教学用书</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_select_other_goods">
      <property name="text">
       <string>搜索其他</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>120</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>我上架的所有书籍</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
